Jenkinsfile

```diff
@@ -68,7 +68,7 @@ pipeline {
                         export TS_CONFIG_MTTCS_DIR=${WORK_HOME}/ts_config_mttcs
 
                         setup -k -r .
-                        pytest --cov-report html --cov=${env.MODULE_NAME} --junitxml=${env.XML_REPORT}
+                        pytest tests/ --cov-report html --cov=${env.MODULE_NAME} --junitxml=${env.XML_REPORT}
```

Comment:
Why are you restricting `pytest` to run only on the `tests` folder? this excludes checking flake8, black, etc from the entire package. I know we are in the process of removing the plugins but even then, I still think it is best is you just run `pytest` on everything.

Reply:
Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!